Sources/MapboxCoreNavigation/Locale.swift

```diff
@@ -24,15 +24,17 @@ extension Locale {
      */
     public static var nationalizedCurrent = Locale(identifier: preferredLocalLanguageCountryCode)
     
+    @available(*, deprecated, message: "Use Locale.current.usesMetricDistanceUnits instead")
     public static var usesMetric: Bool {
-        let locale = self.current as NSLocale
-        guard let measurementSystem = locale.object(forKey: .measurementSystem) as? String else {
-            return false
-        }
-        return measurementSystem == "Metric"
+        Locale.current.usesMetricDistanceUnits
     }
-    
+
+    @available(*, deprecated, renamed: "usesMetricDistanceUnits")
     public var usesMetric: Bool {
+        usesMetricDistanceUnits
+    }
+
+    public var usesMetricDistanceUnits: Bool {
```

Comment:
Nit: “Uses” isn’t a very meaningful word. `prefersMetricDistances`? `measuresDistancesInMetricUnits`?

Reply:
Renamed to `measuresDistancesInMetricUnits` 👍